### PR TITLE
Support for Swift 4.2.1 / Xcode 10.1

### PR DESCRIPTION
**Pull request by RomanPodymov**
Good evening.
Thank you for this library, it is very interesting.
I added support of Swift 4.2.1 / Xcode 10.1. I fixed some warnings. Tested the example on a device with iOS 12.1.1 and it looks good.

### CODE REVIEW

**Comment by mkj-is** [comment]
Sorry we did not make the review sooner, but you forgot to add anyone to reviewers.

File: Example/TFBubbleItUp/AppDelegate.swift [hunk 1]
```diff
@@ -14,30 +14,31 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
 
-    func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
+    func application(_ application: UIApplication,
+                     didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil) -> Bool {
         // Override point for customization after application launch.
         return true
     }
 
-    func applicationWillResignActive(application: UIApplication) {
+    func applicationWillResignActive(_ application: UIApplication) {
```
Comment (mkj-is) [inline Example/TFBubbleItUp/AppDelegate.swift]: The same as in your LocationManager PR, feel free to delete these generated methods.

https://github.com/thefuntasty/LocationManager/pull/9

**Comment by RomanPodymov** [comment]
> Sorry we did not make the review sooner, but you forgot to add anyone to reviewers.

@mkj-is I don't have such permissions, but I hope that I will receive them soon.

File: Example/TFBubbleItUp/AppDelegate.swift (hunk 1 above)
Comment (RomanPodymov) [reply Example/TFBubbleItUp/AppDelegate.swift]: @mkj-is Done.

**Pull request merged**